libs/payments/cart/src/lib/cart.utils.ts

```diff
@@ -1,5 +1,33 @@
-import { CartErrorReasonId } from '@fxa/shared/db/mysql/account';
```

Comment:
[non-blocking] Rather than having a cart.utils.ts that becomes a catchall file, I'd like to have us create a util directory and put each util as it's own file